### ISSUE TITLE
[suggestion] Replace Cure II with Cure I when level synced <30

### ISSUE THREAD

**Issue by pikajude** [MKhayle/XIVComboPlugin#276]
Below level 30, Cure II is unavailable, so there is no decision making involved. This change would allow players who only use Cure II to play synced content without having to change their hotbars, without affecting the experience for players who make use of Freecure.

**Comment by InTheBeef** [comment]
Might the same also apply to Medica 1 & 2?

**Comment by Miral007** [comment]
yes please! same with benefic ii -> benefic, and adloquium -> physick. almost never use the lower level spells in higher content, so they're on less comfy buttons. Would love to have them move to the comfy buttons automatically only when synced.

**Comment by jakekcbyrne** [comment]
#302 I've added this to a PR, up to Attick if he wants to include this or not.
